packages/_private/angular-demo/src/demoform/demoform.component.html

```diff
@@ -0,0 +1,197 @@
+      id="input-password"
+      label="Provide a secure password"
+      name="password"
+      password-toggle
+      pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
+      placeholder="Please provide at least one uppercase and lowercase letter and a number"
+      [required]="true"
+      type="password"
+    />
+
+    <syn-input
+      formControlName="code"
+      id="input-number"
+      label="Please provide a fallback numeric value that may be used for password recovery"
+      min="1000"
+      max="9999"
+      name="code"
+      placeholder="Please choose a value with four digits, e.g. 1234"
+      type="number"
+    />
+  </demo-fieldset>
+  <!-- /Security -->
+
+  <!-- Topics -->
+  <demo-fieldset legend="Topics">
+    <syn-select
+      [clearable]="true"
+      formControlName="topics"
+      id="topics"
+      label="I am interested in the following technologies"
+      [multiple]="true"
+      name="topics"
+      v-model="formData.topics"
+    >
+      <syn-optgroup label="Frontend">
+        <syn-option value="angular">Angular</syn-option>
+        <syn-option value="react">React.js</syn-option>
+        <syn-option value="vanilla">Vanilla.js</syn-option>
+        <syn-option value="vue">Vue</syn-option>
+      </syn-optgroup>
+      <syn-optgroup label="Backend">
+        <syn-option value="node">Node.js</syn-option>
+        <syn-option value="Python">Python</syn-option>
+      </syn-optgroup>
+    </syn-select>
+  </demo-fieldset>
+  <!-- /Topics -->
+
+  <syn-divider />
+
+  <!-- Marketing -->
+  <demo-fieldset legend="Please inform me about the following technologies">
+    <syn-checkbox
+      formControlName="newsletterStandard"
+      id="checkbox-newsletter-default"
+      name="newsletterDefault"
+    >
+      Please subscribe me to the synergy newsletter
+    </syn-checkbox>
+    <syn-checkbox
+      formControlName="newsletterAngular"
+      id="checkbox-newsletter-angular"
+      name="newsletterAngular"
+    >
+      Please subscribe me to all things related to angular
+    </syn-checkbox>
+    <syn-checkbox
+      formControlName="newsletterReact"
+      id="checkbox-newsletter-react"
+      name="newsletterReact"
+    >
+      Please subscribe me to all things related to react
+    </syn-checkbox>
+    <syn-checkbox
+      formControlName="newsletterVanilla"
+      id="checkbox-newsletter-vanilla"
+      name="newsletterVanilla"
+    >
+      Please subscribe me to all things related to vanilla.js
+    </syn-checkbox>
+    <syn-checkbox
+      formControlName="newsletterVue"
+      id="checkbox-newsletter-vue"
+      name="newsletterVue"
+    >
+      Please subscribe me to all things related to vue
+    </syn-checkbox>
+    <syn-switch
+      formControlName="newsletterBeta"
+      id="checkbox-newsletter-beta"
+      name="newsletterBeta"
+    >
+      I am interested in the Synergy Beta Program
+    </syn-switch>
+  </demo-fieldset>
+  <!-- /Marketing -->
+
+  <syn-divider />
+
```

Comment:
Additional information section is missing

Reply:
How did I miss that?